grails-forge-core/src/main/java/org/grails/forge/feature/assetPipeline/AssetPipeline.java

```diff
@@ -135,7 +135,7 @@ public String getCategory() {
 
     @Override
     public String getDocumentation() {
-        return "https://www.asset-pipeline.com/manual/";
+        return "https://github.com/bertramdev/grails-asset-pipeline";
```

Comment:
There is discussion about bringing https://www.asset-pipeline.com back online.
So we can keep this one a bit and see if that materializes.

Also the https://github.com/bertramdev/grails-asset-pipeline is for Grails 2.